### ISSUE TITLE
"Skip to content" link not working

### ISSUE THREAD

**Issue by tasnook** [highcharts/highcharts#16804]
#### Expected behaviour
Activating the Skip to main content link with the enter key should move keyboard focus to the main content on the page.
`<a class="skip-link sr-only sr-only-focusable" href="#content">Skip to content</a>`

#### Actual behaviour
On certain pages that lack an element with `id="content"`, triggering the link does nothing; keyboard focus remains on the link.

#### Live demo with steps to reproduce

1. Go to [Accessibility Statement – Highcharts, (https://www.highcharts.com/blog/article/accessibility-statement/)](https://www.highcharts.com/blog/article/accessibility-statement/)
2. Press the TAB key once; the "Skip to content" link appropriately appears on focus.
3. Press the Enter key; nothing happens, keyboard focus remains on the link

_Screenshot showing focus remaining on Skip to content link in Chrome_
![image](https://user-images.githubusercontent.com/34279228/146617489-91052a05-c9f3-40ea-8d2a-903206b89a70.png)

#### Product version

Webpage: [Accessibility Statement – Highcharts, (https://www.highcharts.com/blog/article/accessibility-statement/)](https://www.highcharts.com/blog/article/accessibility-statement/)

#### Affected browser(s)
Version 85.0.4183.83 (Official Build) (64-bit)

#### NOTE: 
I applaud your accessibility work! That is why I know you would want to know about this bug. Tukk & Skol!

**Comment by oysteinmoseng** [comment]
Thanks for reporting @tasnook, we'll be sure to fix!

**Comment by oysteinmoseng** [comment]
Webteam reported as fixed
